clj-http-lite/project.clj

```diff
@@ -6,6 +6,7 @@
   :plugins [[lein-modules "0.3.11"]]
   :dependencies [[com.github.oliyh/martian :version]
                  [org.martinklepsch/clj-http-lite "0.4.3"]
+                 [clj-commons/clj-yaml "0.7.107"]
```

Comment:
Why not add the clj-yaml dependency to core? Without it, core will not technically compile on its own, and that is where the clj-yaml dependency actually is.

Reply:
Sure. Will do.

I guess I really don't know CLJS and was not sure if I could / should have a CLJ file in something that was CLJC.